relayer/contracts/generate.go

```diff
@@ -1,6 +1,5 @@
 //go:generate bash -c "jq .abi ../../core/packages/contracts/out/OpaqueProof.sol/OpaqueProof.json | abigen --abi - --type OpaqueProof --pkg contracts --out opaque_proof.go"
```

Comment:
Obsolete. No longer needed since we pass the proof directly to `IGateway.submitInbound` without converting it to opaque bytes

```suggestion
```